.circleci/config.yml

```diff
@@ -119,15 +24,33 @@ workflows:
       - release-verification:
           type: approval
           requires:
-            - build
+            - android-sdk/build
           filters:
             tags:
               only: /^v.*/
             branches:
               ignore: /.*/
-      - publish:
+      - android-sdk/publish:
           requires:
             - release-verification
+          post-steps:
+            - run:
+                name: Publish Documentation
```

Comment:
I've also added a config in `android-builconfig` for automatically publishing documentation when `./gradlew publish` is run: https://github.com/rakutentech/android-buildconfig/tree/master/publish/documentation#setup

But probably you can add that sometime later.

Reply:
Cool. I'll create separate PR for that.
Thanks